engine/src/settings.rs

```diff
@@ -188,6 +188,7 @@ pub struct Settings {
 	pub eth: Eth,
```

Comment:
Shouldn't we change this type from `Eth` to `Evm`?

Reply:
I thought that I would do the renaming in another PR. I had opened an issue for it. I added this rename in the issue.
PRO-1221